rest-service/manager_rest/storage/storage_manager.py

```diff
@@ -629,3 +674,59 @@ def __iter__(self):
 
     def __getitem__(self, item):
         return self.items[item]
+
+
+def enable_tracing_for_all_engines():
```

Comment:
maybe export all of this functions to a utils file for tracing

Reply:
This only relates to SQL stuff, you're saure?